community/joe/build

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e 
+    ./configure \
+        --prefix=/usr \
+        --sysconfdir=/etc
```

Comment:
Looks like you ended up with four extra spaces here on these three lines? 

There should also be a blank line after the shebang

Reply:
My bad I had formatting issues on my screen.

Reply:
I wasn't using Joe